src/fenced/fenced/disks.py

```diff
@@ -113,9 +112,13 @@ def __init__(self, fence, name, pool=None):
         self.fence = fence
         self.name = name
         self.pool = pool
-        self.cam = cam.CamDevice(f'/dev/{name}')
         self.curkey = None
 
+        if self.name.startswith('nvd'):
```

Comment:
we could still have is_nvme here
or always have self.cam and self.nvme set to None by default

Comparing startswith() of everytime is repetitive, sorry if you thought thats what i wanted us to do